.github/workflows/fw_icu4c_ci.yml

```
@@ -77,3 +108,19 @@ jobs:
           cd test/cintltst;
           echo 'Running ICU4C cintltst';
           make -j2 check;
+
+  debian-packaging:
+    needs: set-version-number
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Checkout ci-build-tools
+        run: git clone https://github.com/sillsdev/ci-builder-scripts
+
+      - name: Build debian package
+        run: |
+          sudo apt-get install -y devscripts dh-make
```

```suggestion
          sudo apt-get install -y devscripts dh-make dirmngr
          mkdir -v ~/.gnupg
```

I think one or both of these adjustments may be helpful for the recent gha job error.